File: ingredients-catalog-server/src/main/java/com/example/ingredientscatalogserver/ingredient/IngredientDTO.java
package com.example.ingredientscatalogserver.ingredient;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class IngredientDTO {

    private Long id;
    private Long ingredientCode;
    private String ingredientName;
    private Double ingredientCalories;
    private Double ingredientProteins;
    private Double ingredientCarbohydrates;
    private Double ingredientFats;
}

File: MenuServer/MenuServer/Dtos/IngredientDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MenuServer.Dtos
{
    public class IngredientDto
    {
        public int IngredientId { get; set; }
        public string IngredientName { get; set; }
        public int IngerientCode { get; set; }

        public double IngredientCost { get; set; }
        public double IngredientCalories { get; set; }
        public double IngredientProteins { get; set; }
        public double IngredientCarbohydrates { get; set; }
        public double IngredientFats { get; set; }
    }
}
